redbase/implementation/redbase-ex.md

```
@@ -0,0 +1,73 @@
+# The Personal Extension RedBase Part 5: EX
+
+Source: <https://web.stanford.edu/class/cs346/2015/redbase-ex.html>
+
+Proposal Due Sunday, May 17, 11:59PM  
+Implementation Due Friday, June 5
+
+- [Introduction](#intro)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)